### ISSUE TITLE
Linux Mint Support

### ISSUE THREAD

**Issue by tresf** [qzind/qz-print#57]
@klabarge tested the installer and basic functionality and all passed.

The only issues are cosmetic issue with the icon, similar to what we see on Ubuntu and Fedora.

System Tray has white background:
![image](https://cloud.githubusercontent.com/assets/6345473/8469272/c75d3cca-2045-11e5-9f4d-b1dc0cf24fde.png)

Desktop icon isn't correct:
![image](https://cloud.githubusercontent.com/assets/6345473/8469277/02505768-2046-11e5-8564-5fc12070491a.png)

**Comment by tresf** [comment]
@klabarge if you can find where this color is stored :smiling_imp:

**Comment by tresf** [comment]
@klabarge bump

**Comment by tresf** [comment]
**Java**
- First it calls [`getThemeName()`](https://github.com/qzind/qz-print/blob/1.9/src/qz/utils/UbuntuUtilities.java#L90)
- Then it calls [`getTrayColor`](https://github.com/qzind/qz-print/blob/1.9/src/qz/utils/UbuntuUtilities.java#L49)

Mint Cinnamon 17.2 x64 seems to store it around here... (We should also check where Linux Mate stores it)

``` bash
gconftool-2 --get /desktop/cinnamon/windows/theme
# e.g. returns "Mint-X"
cat /usr/share/themes/Mint-X/gtk-3.0/gtk-main.css
# FIXME find out which color is panel
```

Once we get this value, we'll need to determine the best way to get the OS information so that we can switch logic between Ubuntu and Mint.

**Comment by klabarge** [comment]
Linux Mint 17.2 Cinnamon 64-bit

The color for the system tray is stored in the file `/usr/share/themes/Mint-X/cinnamon/cinnamon.css` 

Mint-X is the default theme, so if another theme is chosen, it would fall under its respective `whatever_theme/cinnamon/cinnamon.css` file.  

The actual "Cinnamon" theme however is stored in `/usr/share/cinnamon/theme/cinnamonn.css`

The color is stored in the `#panel` class. The commented out values are the default values.

![image](https://cloud.githubusercontent.com/assets/12505463/9571901/91c13780-4f73-11e5-83af-3c1ec4b2cc09.png)

I will also check on Linux Mint Mate 17.2 64-bit

**Comment by tresf** [comment]
Desktop icon fixed via 3d623ce2817d64489dc5a525507acfe452133608.

Unfortunately the Linux Mint tray icon cannot be fixed easily due to various reasons:
- The themes aren't consistent.  In Mint 17.3 the desktop uses a separate theme from the rest of the components (i.e. [NOT `Mint-X`, but rather one called `Linux Mint`](https://cloud.githubusercontent.com/assets/6345473/11615801/5bfbb230-9c38-11e5-8ee0-e207c8f57daa.png)).
- Even if we assume the correct theme, we'll still run into problems parsing the CSS file by hand.  It's simply not consistent enough for a quick parse like we do with Ubuntu.
  - It uses both [`rgb(255, 255, 255)`](https://cloud.githubusercontent.com/assets/6345473/11615826/f66c0662-9c38-11e5-9ab2-1a2790af2a01.png)
    and `#FFFFFF` style colors
  - Panel colors [don't have unique names](https://cloud.githubusercontent.com/assets/6345473/11615826/f66c0662-9c38-11e5-9ab2-1a2790af2a01.png), so parsing requires first finding the `#parent { /* */ }` CSS component by ID.

Closing as wont-fix for now as the amount of work to change the icon color for Mint is not worth it at this time.

.
